dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -24,6 +24,26 @@ _get_cluster_status(){
     echo $(rosa list clusters -o json | jq -r '.[] | select(.name == '\"$1\"') | .status.state')
 }
 
+_wait_for_cluster_ready(){
+    echo "INFO: Waiting about 1 hour for cluster on ready status"
+    # 180 iterations, sleeping 60 seconds, 3 hours of wait
+    for i in `seq 1 180` ; do
+        CLUSTER_STATUS=$(_get_cluster_status $1)
+        if [ ${CLUSTER_STATUS} == "ready" ] ; then
+            sleep 60 # Wait 60 seconds to allow OCM to really finish the cluster installation
+	    echo "INFO: Cluster on ready status, dumping installation logs..."
+            rosa logs install -c $1
+            return 0
+        else
+            echo "INFO: ${i}/60. Cluster on ${CLUSTER_STATUS} status, waiting 60 seconds for next check"
+            sleep 60
+        fi
+    done
+    echo "ERROR: Cluster $1 not installed after 3 hours, dumping installation logs..."
```

Comment:
OK, was wondering if this could be shorter maybe ~2hrs but 3 is fine. 

Do we use a different job to scale up? Then we should be fine setting lower timeout for failure since we would always only deploy smaller clusters to begin with.